packages/http/src/mocker/generator/JSONSchema.ts

```diff
@@ -32,13 +32,38 @@ export function generate(bundle: unknown, source: JSONSchema): Either<Error, unk
     stripWriteOnlyProperties(source),
     E.fromOption(() => Error('Cannot strip writeOnly properties')),
     E.chain(updatedSource =>
-      // necessary as workaround broken types in json-schema-faker
-      // @ts-ignore
-      tryCatch(() => jsf.generate({ ...cloneDeep(updatedSource), __bundled__: bundle }), toError)
+      tryCatch(
+        // necessary as workaround broken types in json-schema-faker
+        // @ts-ignore
+        () => sortSchemaAlphabetically(jsf.generate({ ...cloneDeep(updatedSource), __bundled__: bundle })),
+        toError
+      )
     )
   );
 }
 
+//sort alphabetically by keys
+export function sortSchemaAlphabetically(source: any): any {
+  if (Array.isArray(source)) {
+    for (const i of source) {
+      if (typeof source[i] === 'object') {
+        source[i] = sortSchemaAlphabetically(source[i]);
+      }
+    }
+    return source;
+  } else if (typeof source === 'object') {
+    Object.keys(source).forEach((key: string) => {
+      if (typeof source[key] === 'object') {
+        source[key] = sortSchemaAlphabetically(source[key]);
+      }
+    });
+    return Object.fromEntries(Object.entries(source).sort());
+  }
```

Comment:
fixed. [0de8272](https://github.com/stoplightio/prism/pull/2041/commits/0de82721da89e3601cf0de688fcb5a97e570acdb)

Reply:
enter a whole debate on why it's there (because the spec says son) and why people believe that's a mistake: https://stackoverflow.com/questions/18808226/why-is-typeof-null-object 😂